helo.js

```diff
@@ -0,0 +1,123 @@
+// You are talking to a friend who suddently looks at you
```

Comment:
Is there any difference between the ﻿`exercise1` file and ﻿`helo.js`? If not, you can delete the duplicate file. :)

Reply:
yes, its dublicates.

Reply:
Then we can remove the file :)